script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -864,7 +864,7 @@ Plowing her way across the glacier, she closes in on the immobile vampire.
 -- Page 629, Offset 12513.
 -- 「“女――――――！”」
 // 女!
-"Woman――――――!"
+"Woman―――!"
```

Comment:
Vlov has old sensibilities, so maybe it's fine for him to say this but; lol

Obviously the intent is more as a curse word or something.

Reply:
Yeah I dunno, if he said 'wench!' here that would probably feel equally ridiculous

Reply:
Uh, yes. But I was thinking something like "Damn you!" or something. Well, this one isn't as bad off as the below.